shared-bindings/board/__init__.c

```diff
@@ -73,6 +73,30 @@ mp_obj_t board_i2c(void) {
 MP_DEFINE_CONST_FUN_OBJ_0(board_i2c_obj, board_i2c);
 
 
+//| def SECOND_I2C() -> busio.I2C:
+//|     """Returns the `busio.I2C` object for the secondary I2C pins, usually the STEMMA[QT] connector. It is a singleton."""
+//|     ...
+//|
```

Comment:
There is no board right now where it's called `SECOND_I2C`, so let's add `STEMMA_I2C` to the doc instead. I've added some language to the doc to explain that it might not be present.
```suggestion
//| def STEMMA_I2C() -> busio.I2C:
//|     """Returns the `busio.I2C` object for the STEMMA or STEMMA QT connector.
//|     Present only if the STEMMA connector provides a second, additional I2C bus that is not the same as `board.I2C()`.
//|     It is a singleton."""
//|     ...
//|
```